head.html

```diff
@@ -3,6 +3,7 @@
 <link rel="preload" href="/nav.plain.html" as="fetch" crossorigin="anonymous" />
 <script src="/scripts/lib-franklin.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
+<script async src="https://app.enzuzo.com/apps/enzuzo/static/js/__enzuzo-cookiebar.js?uuid=04cbcc9c-62ba-11ee-b344-331207534791"></script>
```

Comment:
@Lauraa1003  Can you try moving it into delayed.js ?

Reply:
It's not working anymore inside delayed.js..